tests/syscall/mod.rs

```diff
@@ -137,7 +137,7 @@ fn get_att() {
     run_test(bin, 0, None, None, None);
 }
 
-#[cfg(feature = "backend-sgx")]
+#[cfg(enarx_with_shim)]
```

Comment:
Already guarded in `test/integration.rs`

Reply:
removed